rotkehlchen/constants/assets.py

```diff
@@ -2209,3 +2209,15 @@
     protocol=None,
 )
 CONSTANT_ASSETS.append(A_ENS)
+A_PSP = EthereumToken.initialize(
+    address=string_to_ethereum_address('0xcAfE001067cDEF266AfB7Eb5A286dCFD277f3dE5'),
+    decimals=18,
+    name='ParaSwap',
+    symbol='PSP',
+    started=Timestamp(1636966698),
+    swapped_for=None,
+    coingecko='paraswap',
+    cryptocompare=None,
+    protocol=None,
+)
+CONSTANT_ASSETS.append(A_ENS)
```

Comment:
This is wrong. Should be CONSTANT_ASSETS.append(A_PSP)

Reply:
ah yes indeed. Please do fix. Sorry did not see it.